templates/logosSection.hbs

```diff
@@ -0,0 +1,16 @@
+<section class="sponsors-wrapper">
+  <div class="sponsors-section">
+    <div class="sponsors-section__eyebrow">
+      <p class='eyebrow-typography sponsors-section__eyebrow--title'>{{eyebrow}}</p>
+      <h3 class='heading-3 sponsors-section__eyebrow--description'> {{title}}</h3>
+    </div>
+    <div class='sponsors-section__logos'>
+      {{#each logos}}
+        <img class="sponsors-section__logos--img" src="{{assetSrc this}}" alt="{{assetTitle this}}" />
+      {{/each}}
+    </div>
+  </div>
+
+  <img class='sponsors-wrapper__shape-a' src="/image/Shape-A.svg">

```

Comment:
Please add the alt atribute empty like this `alt=""` for this and the images below.